salt/metalk8s/repo/installed.sls

```diff
@@ -29,7 +29,7 @@ Generate repositories nginx configuration:
 Deploy container registry nginx configuration:
   file.managed:
     - name: {{ nginx_registry_path }}
-    - source: salt://{{ slspath }}/files/{{ nginx_registry }}
```

Comment:
Why this change? Is `{{ nginx_registry }}` now unused?

Reply:
It's used to compute the path `nginx_registry_path`